make.paws/tests/testthat/test_sdk_helper.R

```diff
@@ -0,0 +1,306 @@
+test_that("check list paws packages", {
+  tmp_dir <- tempdir()
+  dir_list <- c("paws", "paws.cat1", "paws.cat2", "fake")
+  sapply(file.path(tmp_dir, dir_list), dir.create)
+
+  expect_equal(
+    basename(list_paws_pkgs(tmp_dir)),
+    c("paws", "paws.cat1", "paws.cat2")
+  )
+  fs::dir_delete(file.path(tmp_dir, dir_list))
+})
+
+test_that("check list paws packages restrict package", {
+  tmp_dir <- tempdir()
+  dir_list <- c("paws", "paws.cat1", "paws.cat2", "fake")
+  sapply(file.path(tmp_dir, dir_list), dir.create)
+
+  expect_equal(
+    basename(list_paws_pkgs(tmp_dir, "paws.cat1")),
```

Comment:
Why is `basename` needed here?

Reply:
`list_paws_pkgs` will return the parent directory so something like:  `file.path(tmp_dir, "paws.cat1")`. I will update the test to reflect this :)